scripts/release/update-mgit-branches.js

```diff
@@ -18,12 +18,12 @@ if ( !branchName ) {
 const path = require( 'path' );
```

Comment:
I forgot to rename this file…

Reply:
Not a problem. We're removing it very soon.